src/pyhf/readxml.py

```diff
@@ -1,8 +1,20 @@
 from __future__ import annotations
 
 import logging
-import os
-from typing import TYPE_CHECKING, Callable, Iterable, Tuple, Union, IO
+from typing import (
+    Callable,
+    Iterable,
+    Tuple,
+    IO,
+    List,
+    Set,
+    Union,
+    Sequence,
+    MutableMapping,
+    MutableSequence,
+    cast,
+)
```

Comment:
This is just a suggestion to avoid more import movement noise if we end up eventually turning on `isort` in `pre-commit`. Feel free to ignore though.

```suggestion
from typing import (
    IO,
    Callable,
    Iterable,
    List,
    MutableMapping,
    MutableSequence,
    Sequence,
    Set,
    Tuple,
    Union,
    cast,
)
```